book/src/01_intro.md

```diff
@@ -0,0 +1,83 @@
+# Introduction
+
+Welcome to The Specs Book, an introduction into ECS and the Specs API.
+This book is targeted to beginners, taking you through all the difficulties
+from setting up to building and structuring a game with an ECS.
+
+Specs is an ECS library allowing parallel system execution, with both low
+overhead and high flexibility, different storage types and a type-level
+system data model.
+
+You didn't fully understand what this sentence was about? The next section
+is for you! In case you already know what an ECS is, just skip it.
+
+## What's an ECS?
+
+The term **ECS** is a shorthand for Entity-component system. These are the three
+core concepts. Each **entity** is associated with some **components**. Those entities and
+components are processed by **systems**. This way, you have your data (components)
+completely separated from the behaviour (systems). The entity just logically
+groups components, so a `Velocity` is applied to the `Position` of the
+same entity.
+
+ECS is sometimes seen as a counterpart to Object-Oriented Programming. I wouldn't
+say that's a hundred percent true, but let me give you some comparision here.
+
+In OOP, your player might look like this (I've used Java for the example):
+
+```java,ignore
+public class Player extends Character {
+    private final Transform transform;
+    private final Inventory inventory;
+}
+```
+
+There are several limitations here:
+
+* There is no multiple inheritance, therefore OOP programming is limited to
+  extending just `Character`; moreover, you have to think about "*Is* the player
+  a collider or *has* it a collider?"
+* You cannot easily extend the player with modding; all the
```

Comment:
do we really need line breaks in the book? any editor will do them automatically, and better